azure-pipelines.yml

```diff
@@ -131,6 +133,7 @@ stages:
           condition: ne(variables['SkipTests'], 'true')
           env:
             RunIntegrationTests: $(RunIntegrationTests)
+            POCKETLOGGER_LOG_PATH: $(PocketLoggerLogPath)
```

Comment:
We need this set as well for the unit tests as well as for the child process (the `AspNetProcess` class) spawned by the integration tests.

Setting it globally is really the simplest approach.

Reply:
Cool, my latest push should handle setting that more globally.